### PR TITLE
Error response follows Token Error Response + additional error codes for authorization data erros

### PR DESCRIPTION
**Pull request by tlodderstedt**
related to #23

### CODE REVIEW

File: main.md [hunk 1]
```diff
@@ -219,24 +219,26 @@ The following is an example of such a response:
...
+In addition to the error codes as defined in Section 5.2 of [@!RFC6749], the pushed authorization request endpoint will also respond with the following HTTP status codes, and, if necessary, error codes. 
 
-#### Authorization required
-If the client is not authorized to perform the authorization request it wanted to post, the authorization server shall return `403 Forbidden` HTTP error response.
+### Unsupported Response Type
+The authorization server does not support obtaining an authorization code using this method. The AS responds with HTTP status code 400 and `error` value `unsupported_response_type`.
 
-#### Invalid request
-If the request object received is invalid, the authorization server shall return `400 Bad Request` HTTP error response.
+### Invalid Scope
```
Comment (b---c) [inline main.md]: invalid_scope is already in Section 5.2 of [@!RFC6749] FWIW

File: main.md [hunk 2]
```diff
@@ -219,24 +219,26 @@ The following is an example of such a response:
...
+The requested scope is invalid, unknown, or malformed. The AS responds with HTTP status code 400 and `error` value `invalid_scope`. 
+              
+### Invalid Redirect URI
+The requested scope is invalid, unknown, or malformed. The AS responds with HTTP status code 400 and `error` value `invalid_redirect_uri`.
```
Comment (jogu) [inline main.md]: Looks like a C&P error, 'The requested scope is invalid, unknown, or malformed.' should presumably say something like 'invalid or mismatching redirection URI'.

File: main.md (hunk 1 above)
Comment (tlodderstedt) [reply main.md]: thanks. removed it.

File: main.md (hunk 2 above)
Comment (tlodderstedt) [reply main.md]: thanks. fixed it.

**Pull request merged**